keywords.txt

```diff
@@ -282,10 +288,14 @@ MY_ESP8266_SSID	LITERAL1
 # MY_DEBUG_VERBOSE_CORE
 # MY_GATEWAY_FEATURE
 # MY_RAM_ROUTING_TABLE_ENABLED
+# MY_RF24_CONFIGURATION
+# MY_RFM69HW
```

Comment:
internal define

Reply:
@tekka007 did you see the comment at top of this block? https://github.com/mfalkvidd/MySensors/blob/67bccd15b8feda9c94a3267b968ad5e32bff68f1/keywords.txt#L278